### PR TITLE
Update "Kafka Connector Example" Doc

### PR DESCRIPTION
**Pull request by chathuranga-jayanath-99**
## Purpose
This PR includes updates to the below mentioned page.
https://mi.docs.wso2.com/en/4.3.0/reference/connectors/kafka-connector/kafka-connector-producer-example/

The updated page will appear as shown in the attached PDF below.
[Kafka Example - WSO2 Micro Integrator Documentation 4.3.0.pdf](https://github.com/user-attachments/files/16258951/Kafka.Example.-.WSO2.Micro.Integrator.Documentation.4.3.0.pdf)

### CODE REVIEW

File: en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md [hunk 1]
```diff
@@ -16,40 +16,68 @@ If you do not want to configure this yourself, you can simply [get the project](
...
+    3. Use following values can to create the connection. 
+        - Connection Name - KafkaConnection
+        - Connection Type - kafka
+        - Bootstrap Servers - localhost:9092 
+        - Key Serializer Class - org.apache.kafka.common.serialization.StringSerializer
+        - Value Serializer Class - org.apache.kafka.common.serialization.StringSerializer
+        - Pooling Enabled - false
 
-    <a href="{{base_path}}/assets/img/integrate/connectors/filecon10.png"><img src="{{base_path}}/assets/img/integrate/connectors/filecon10.png" title="working directory" width="800" alt="working directory"/></a>
+## Creating the Integration Logic
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md:38]: Use sentence case and avoid `ing` in verbs for titles. https://learn.microsoft.com/en-us/style-guide/text-formatting/formatting-titles

Check and fix all places.

File: en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md [hunk 2]
```diff
@@ -16,40 +16,68 @@ If you do not want to configure this yourself, you can simply [get the project](
 
 ## Set up Kafka
 
-Before you begin, set up Kafka by following the instructions in [Setting up Kafka]({{base_path}}/reference/connectors/ksfka-connector/setting-up-kafka/).
+Before you begin, set up Kafka by following the instructions in [Setting up Kafka]({{base_path}}/reference/connectors/kafka-connector/setting-up-kafka/).
 
-## Configure the connector in WSO2 Integration Studio
+## Setting up the Integration Project
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md:21]:
``` suggestion
## Set up the integration project
```

File: en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md [hunk 3]
```diff
@@ -16,40 +16,68 @@ If you do not want to configure this yourself, you can simply [get the project](
...
+            <faultSequence>
+            </faultSequence>
         </resource>
-    </api>
+	</api>
     ```
 Now we can export the imported connector and the API into a single CAR application. The CAR application needs to be deployed during server runtime. 
 
-{!includes/reference/connectors/exporting-artifacts.md!}
+## Exporting Integration Logic as a CApp
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md:78]:
```suggestion
## Export integration logic as a carbon application 
```

File: en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md [hunk 4]
```diff
@@ -16,40 +16,68 @@ If you do not want to configure this yourself, you can simply [get the project](
...
 
-    <a href="{{base_path}}/assets/img/integrate/connectors/filecon10.png"><img src="{{base_path}}/assets/img/integrate/connectors/filecon10.png" title="working directory" width="800" alt="working directory"/></a>
+## Creating the Integration Logic
 
+1. Select Micro Integrator and click on `+` in APIs to create a REST API. Provide `KafkaTransport` as name and `publishMessages` as context.
+   <img src="{{base_path}}/assets/img/integrate/connectors/kafka-conn-add-api.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>
 
-1. Right click on the created Integration Project and select **New** -> **Rest API** to create the REST API.
+2. Then create a resource with the below configuration.
+
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md:44]: extra new line?

File: en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md [hunk 5]
```diff
@@ -16,40 +16,68 @@ If you do not want to configure this yourself, you can simply [get the project](
...
+1. Follow the steps in [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up the Integration Project.
 
-{!includes/reference/connectors/importing-connector-to-integration-studio.md!}
+2. Create a new Kafka connection. 
+    1. Goto `Local Entries` -> `Connections` and click on the `+` sign. 
+    2. Select  `KafkaTransport` connector.
+        <img src="{{base_path}}/assets/img/integrate/connectors/kafka-conn-add-new-connection.png" title="Add new kafka connection" width="800" alt="Add new kafka connection"/>
 
-5. Create a new Kafka connection by selecting a particular operation.
+    3. Use following values can to create the connection.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md:30]:
```suggestion
    3. Use the following values to create the connection. 
```

File: en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md [hunk 6]
```diff
@@ -16,40 +16,68 @@ If you do not want to configure this yourself, you can simply [get the project](
...
+        - Pooling Enabled - false
 
-    <a href="{{base_path}}/assets/img/integrate/connectors/filecon10.png"><img src="{{base_path}}/assets/img/integrate/connectors/filecon10.png" title="working directory" width="800" alt="working directory"/></a>
+## Creating the Integration Logic
 
+1. Select Micro Integrator and click on `+` in APIs to create a REST API. Provide `KafkaTransport` as name and `publishMessages` as context.
+   <img src="{{base_path}}/assets/img/integrate/connectors/kafka-conn-add-api.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>
 
-1. Right click on the created Integration Project and select **New** -> **Rest API** to create the REST API.
+2. Then create a resource with the below configuration.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md:43]:
```suggestion
2. Then, create a resource with the below configuration.
```

Comment (anupama-pathirage) [inline en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md:43]:
```suggestion
2. Create a resource with the below configuration.
```

File: en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md [hunk 7]
```diff
@@ -16,40 +16,68 @@ If you do not want to configure this yourself, you can simply [get the project](
...
+    
+    <img src="{{base_path}}/assets/img/integrate/connectors/kafka-conn-add-resource.png" title="Adding API Resource" width="400" alt="Adding API Resource"/>
+
+3. Select the created resource and add `PublishMessages` operation.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md:47]:
```suggestion
3. Select the created resource and add the `PublishMessages` operation.
```

File: en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md [hunk 8]
```diff
@@ -16,40 +16,68 @@ If you do not want to configure this yourself, you can simply [get the project](
...
+    <img src="{{base_path}}/assets/img/integrate/connectors/kafka-conn-add-operation.png" title="Adding operation" width="800" alt="Adding operation"/>
+
+    - Use following values to fill the appearing form.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md:50]:
```suggestion
    - Use the following values to fill the appearing form.
```

File: en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md [hunk 9]
```diff
@@ -16,40 +16,68 @@ If you do not want to configure this yourself, you can simply [get the project](
...
+        - Connection - KafkaConnection
+        - Topic - test
+        - Partition Number - 0
 
-2. Specify the API name as `KafkaTransport` and API context as `/publishMessages`. You can go to the source view of the XML configuration file of the API and copy the following configuration (source view).
+        <img src="{{base_path}}/assets/img/integrate/connectors/kafka-conn-config-operation.png" title="Configure operation" width="400" alt="Configure operation"/>
+
+Source view of the XML configuration file of the API will be as below.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md:57]:
```suggestion
The source view of the XML configuration file of the API will be as below.
```

File: en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md [hunk 10]
```diff
@@ -16,40 +16,68 @@ If you do not want to configure this yourself, you can simply [get the project](
...
             </inSequence>
-            <outSequence/>
-            <faultSequence/>
+            <faultSequence>
+            </faultSequence>
         </resource>
-    </api>
+	</api>
     ```
 Now we can export the imported connector and the API into a single CAR application. The CAR application needs to be deployed during server runtime.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md:76]:
```suggestion
Now, we can export the imported connector and the API into a single CAR application. The CAR application needs to be deployed during server runtime. 
```

File: en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md [hunk 11]
```diff
@@ -14,42 +14,69 @@ The following diagram illustrates all the required functionality of the Kafka se
...
+                    <valueSchemaSoftDeleted>false</valueSchemaSoftDeleted>
+                </kafkaTransport.publishMessages>
             </inSequence>
-            <outSequence/>
-            <faultSequence/>
+            <faultSequence>
+            </faultSequence>
         </resource>
-    </api>
+	</api>
```
Comment (chanikag) [inline en/docs/reference/connectors/kafka-connector/kafka-connector-producer-example.md:73]: Check the formatting.

**Review by chanikag — APPROVED** [review]

**Pull request merged**